docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx

```diff
@@ -530,6 +527,74 @@ be replaced with a sheet of paper icon like so: <img src="/img/outdoornav_images
 
 :::
 
+### Points of Interest
+
+Users are now able to add Points of Interest (POI) onto their map, marking a location or and object that is persistent
+across both "Map" and "Waypoint" planning modes. The intended use for the points of interest feature is to senable the user to
+set up locations that they know will almost never change and be able to for example navigate, inspect these locations.
+
+#### Creating POIs
+
+To create a POI, ensure that you are in "POI edit mode" (Hotkey, e+4). Click on the map to place your POI and assign
+the POI a name. You also have the ability on creation to assign a heading, tags and custom fields (all optional).
+
+<center>
+  <figure>
+    <img src="/img/outdoornav_images/ui_poi_edit.png" />
+    <figcaption>Point of Interest creation</figcaption>
+  </figure>
+</center>
+
+#### Editing POIs
+
+To edit an existing POI, right click the POI and select Edit. This will allow you to modify any of the existing properties
+of the POI. If you want to simply modify the heading of a POI, you can use the CTRL hotkey to enable heding modification
+and rotate the POI to your dsired heading. See video below for an example of the editing the heading of a POI using the 
+(CTRL hotkey).
+
+<center>
+  <figure>
+    <video autoplay="true" loop="true">
+      <source src="/video/outdoornav_videos/ui_rotating_poi.mp4"/>
+    </video>
+    <figcaption>Rotating a Point of Interests heading</figcaption>
+  </figure>
+</center>
+
+#### Predefined actions
+
+If a POI is assigned one of the two predefined tags, `goto` and/or `inspect`, the user will have the ability to perform
+some quick actions when they right click the POI. In "Map mode" only, not in "Waypoint mode", if the POI has a `goto` tag, 
+the user will see a `Go To` option in the right click options that will autonomously navigate the UGV to the location of 
+the POI (if it is within the bounds of the current network). If the POI has an `inspect` tag, the user will see an `Inspect`
+option in the right click options that will move the PTZ camera to view the POI when clicked. See the below image for an
+example of a POI that has both the `goto` and `inspect` tags, and therefore has the `Go To` and `Inspect` action options
+when right clicked.
+
+<center>
+  <figure>
+    <img src="/img/outdoornav_images/ui_poi_actions.png" />
+    <figcaption>Point of Interest actions</figcaption>
+  </figure>
+</center>
+
+#### Creating Missions from POIs
+
+If you, for example, have a patrol application in which you want your UGV to always navigate to a set of predefined locations.
+You can do such a thing by creating POIs for each of the stops that the UGV should make. The by entereing the network mission
+edit mode (hotkey: e+3), the user can simply click on the POIs in order of traversal to create your network mission. The network 

```

Comment:
We're switching back and forth between 'Your' and 'user' here. I'd say maybe try and keep it all as 'user' based point of view.